<img src="https://github.com/saezlab/dorothea/raw/master/man/figures/tool_logo.png" align="left" height="100">

# DoRothEA 

<br />

<!-- badges: start -->
![GitHub](https://img.shields.io/github/license/saezlab/dorothea)
<!-- badges: end -->

[DoRothEA](https://saezlab.github.io/dorothea/) is a gene regulatory network linking Transcription Factors (TFs) with their downstream targets. The unity of TF 
and its targets is referred as regulon. Such gene regulatory networks allow the computation of TF-activities from gene expression data by enrichment analysis 
using the regulons as underlying gene-sets. The network has been build integrating TF-target interactions from four different strategies: (i) manually curated 
interaction repositories, (ii) interactions derived from ChIP-seq binding data, (iii) in silico predictions of TF binding on gene promoters, and (iv) 
reverse-engineered regulons from large gene expression datasets. Based on the number of supporting evidences each interaction is accompanied with a confidence 
level ranging from high-confident (A) to low-confident (E).